lib/Search/UI/SearchFormView.php

```diff
@@ -56,7 +56,7 @@ class SearchFormView extends View
 {
     public function __construct()
     {
-        parent::__construct(__DIR__ . '/templates/search.form.tpl');
+        parent::__construct(get_custom_file_if_exists(__DIR__ . '/templates/search.form.tpl'));
```

Comment:
not sure `get_custom_file_if_exists` will work when using __DIR__ as this will include the the complete path e.g `/var/www/SuiteCRM`, so it would likely require a full path to use this function here

Reply:
Maybe `parent::__construct(__DIR__ .  get_custom_file_if_exists('/templates/search.form.tpl'));` would work, no?
Do we ever change the working directory, or do we always stay at the SuiteCRM root directory?

Reply:
Updated to a tweaked version of pgorods suggestion.